### ISSUE TITLE
Script doesn't include new libraries folder in latest Bela version

### ISSUE THREAD

**Issue by ebai101** [AndrewCapon/OSXBelaCrossCompiler#2]
Since Bela has encapsulated libraries like Midi and Scope in the new "libraries" folder, they are no longer in `/root/Bela/include`. Therefore this script doesn't copy these libraries into the sysroot clone.

A quick fix: `cp -r /.../bela_repo/libraries /usr/local/linaro/BelaSysroot/root/Bela/include`

**Comment by AndrewCapon** [comment]
Does everything else still work after doing this?

**Comment by ebai101** [comment]
Yes, everything seems to work fine. I'm not actually using Eclipse, I'm in VSCode - but this fixed the problem for me. Probably should confirm that it works for Eclipse.

**Comment by AndrewCapon** [comment]
Super, thanks very much for the info.

I need to put some time aside to update to the new version of Bela and make sure the eclipse stuff still works.

I will look to update the script tomorrow...
